clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -267,6 +272,12 @@
                                         atLeast(0L),
                                         Importance.LOW,
                                         CommonClientConfigs.RECONNECT_BACKOFF_MS_DOC)
+                                .define(RECONNECT_BACKOFF_MAX_CONFIG,
+                                        Type.LONG,
+                                        50L,
```

Comment:
Default to a max of 50ms?  These seems pretty short.  I would think a max of 1000ms is reasonable?